kobo/django/xmlrpc/models.py

```diff
@@ -14,3 +14,6 @@ class XmlRpcLog(models.Model):
 
     def __unicode__(self):
         return u"%s: %s" % (self.user, self.method)
+
+    def __str__(self):
+        return u"%s: %s" % (self.user, self.method)
```

Comment:
`__str__` is meant to return `str` on python2 and not unicode:

```suggestion
        return "%s: %s" % (self.user, self.method)
```